tools/harness-thci/OpenThread_WpanCtl.py

```diff
@@ -78,10 +77,10 @@ def __init__(self, **kwargs):
             self.AutoDUTEnable = False
             self._is_net = False                  # whether device is through ser2net
             self.logStatus = {'stop':'stop', 'running':'running', 'pauseReq':'pauseReq', 'paused':'paused'}
-            self.joinStatus = {'notstart':'idle', 'ongoing':'discover', 'succeed':'joined', 'failed':'failed'}
             self.logThreadStatus = self.logStatus['stop']
```

Comment:
should all `logStatus` and `logThreadStatus` be removed ? e.g. Line#79, #80 and #197~200